x-pack/solutions/security/plugins/cloud_security_posture/server/routes/graph/parse_records.ts

```diff
@@ -151,31 +179,59 @@ const createNodes = (records: GraphEdge[], context: Omit<ParseContext, 'edgesMap
   }
 };
 
-const determineEntityNodeShape = (
+const determineEntityNodeVisualProps = (
```

Comment:
return type is not accurate, I believe you miss declaration of `label` and `tag`